packages/controlled-vocabulary/package.json

```diff
@@ -12,8 +12,8 @@
     "build": "webpack --mode production && flow-copy-source -v src types"
   },
   "dependencies": {
-    "@performant-software/semantic-components": "^1.1.3",
-    "@performant-software/shared-components": "^1.1.3",
+    "@performant-software/semantic-components": "^1.2.0-beta.5",
```

Comment:
Thinking about naming things: since other packages are named for their function, maybe we rename semantic components and shared components? Let's discuss what to call them. An idea for semantic components, based on [this architectural style](https://www.architectural-review.com/essays/books/heroic-concrete-architecture-and-the-new-boston): "Heroic Library". 💭